src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -72,13 +73,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
         .map { case (root, f) => (root, stripExtensionIfModuleFile(f)) }
         .filter { case (root, f) => fileNameFilter(f.getName) }
         .map(relativize)
+        .map(fixRelativePathForWindows)
         .map(relativeFileToLoadPath)
 
     def relativize(baseAndFile: (File, File)): String = {
       val (base, file) = baseAndFile
       base.toURI().relativize(file.toURI()).getPath()
     }
 
+    def fixRelativePathForWindows(path: String): String =
+      if (runningOnWindows) path.replace('/', File.separatorChar)
```

Comment:
You are absolutely right. 

--
Another idea:
Java on Windows can handle `/` as a file separator, maybe this is a better solution than using `File.separatorChar`. With this solution the replacing of file separators won't be necessary at all.